Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/) for ChainSafe Files/Chainsafe Files user interface.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/chainsafe-files/-/chainsafe-files-user-interface/horizontal-auto.svg)

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 44.9% (128 of 285 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/nb_NO/

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ChainSafe/ui-monorepo?pullRequest=1373) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ChainSafe/ui-monorepo?pullRequest=1373) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: comradekingu<br/>:x: weblate<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ChainSafe/ui-monorepo?pullRequest=1373) it.</sub>

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

Commit:
Translated using Weblate (French)

Currently translated at 99.6% (285 of 286 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

Tbaut committed:
Merge branch 'dev' into weblate-chainsafe-files-chainsafe-files-user-interface
